subiquity/server/controllers/install.py

```diff
@@ -339,6 +339,13 @@ async def postinstall(self, *, context):
                 ubuntu_drivers = self.app.controllers.Drivers.ubuntu_drivers
                 await ubuntu_drivers.install_drivers(root_dir=self.tpath(),
                                                      context=child)
+        if self.model.codecs.do_install:
+            # NOTE currently, ubuntu-restricted-addons is an empty package that
+            # pulls relevant packages through Recommends:
+            # Ideally, we should make sure to run the APT command with
+            # the --install-recommends option.
+            await self.install_package(context=context,
```

Comment:
Instead of doing this here you can implement target_packages on CodecsModel

Reply:
Oh, right, that would be much cleaner. Thanks